### PR TITLE
BigInt

### PR DESCRIPTION
**Pull request by amirhshahbazi**
Translated the BigInt page.

### CODE REVIEW

**Review by mahdyar — CHANGES_REQUESTED** [review]
You don't need to make a new pull request for any changes you make. you can add more commits by pushing to the `BigInt` branch on your fork.
In addition to that, please read the [`Translation Tips`](https://github.com/javascript-tutorial/fa.javascript.info#translation-tips):
> Please keep line breaks and paragraphs "as is": don't add new lines and don't remove existing ones. Makes it easy to merge future changes from the English version into the translation.

You've removed the newline on line 8, please add it back.

**Comment by amirhshahbazi** [comment]
Thanks for the review, @mahdyar
/done

File: 1-js/99-js-misc/05-bigint/article.md [hunk 1]
```diff
@@ -2,129 +2,129 @@
...
 const bigint = 1234567890123456789012345678901234567890n;
 
 const sameBigint = BigInt("1234567890123456789012345678901234567890");
 
-const bigintFromNumber = BigInt(10); // same as 10n
+const bigintFromNumber = BigInt(10); // با 10n یکسان است
 ```
 
-## Math operators
+## عملیات ریاضی
```
Comment (mahdyar) [inline 1-js/99-js-misc/05-bigint/article.md:17]: `Math operators` means `عملگرهای ریاضی`, not `عملیات ریاضی`.

File: 1-js/99-js-misc/05-bigint/article.md [hunk 2]
```diff
@@ -2,129 +2,129 @@
...
+یک `BigInt` را می توان عمدتا به صورت یک عدد عادی استفاده کرد، برای مثال:
 
 ```js run
 alert(1n + 2n); // 3
 
 alert(5n / 2n); // 2
 ```
 
-Please note: the division `5/2` returns the result rounded towards zero, without the decimal part. All operations on bigints return bigints.
+در نظر داشته باشید: عملیات تقسیم 5/2 نتیجه را بدون قسمت اعشار و رند شده به سمت صفر، باز می گرداند. همه ی عملیات بر روی bigint ها، مقداری را از نوع bigint باز می گردانند.
```
Comment (mahdyar) [inline 1-js/99-js-misc/05-bigint/article.md:27]: Put `5/2` between backticks as it was.

File: 1-js/99-js-misc/05-bigint/article.md [hunk 3]
```diff
@@ -2,129 +2,129 @@
...
 
-We can't mix bigints and regular numbers:
+اعداد نوع bigint را نمی توان با اعداد معمولی مخلوط کرد:
 
 ```js run
-alert(1n + 2); // Error: Cannot mix BigInt and other types
+alert(1n + 2); // خطا: Cannot mix BigInt and other types
```
Comment (mahdyar) [inline 1-js/99-js-misc/05-bigint/article.md:32]: There is no need to translate `Error` here.

File: 1-js/99-js-misc/05-bigint/article.md [hunk 4]
```diff
@@ -2,129 +2,129 @@
...
 ```
 
-We should explicitly convert them if needed: using either `BigInt()` or `Number()`, like this:
+می بایست آن ها ابتدا به صورت زیر، توسط `BigInt()` یا `Number()` تبدیل کرد:
```
Comment (mahdyar) [inline 1-js/99-js-misc/05-bigint/article.md:35]: You missed `if needed`.

File: 1-js/99-js-misc/05-bigint/article.md [hunk 5]
```diff
@@ -2,129 +2,129 @@
...
 
-alert( +bigint ); // error
+alert( +bigint ); // خطا
 ```
-So we should use `Number()` to convert a bigint to a number.
+باید ابتدا از `Number()` برای تبدیل یک bigint به نوع عددی استفاده کرد.
 ````
 
-## Comparisons
+## عملگرهای مقایسه
```
Comment (mahdyar) [inline 1-js/99-js-misc/05-bigint/article.md:62]: `مقایسه‌ای` would be a better translation.

File: 1-js/99-js-misc/05-bigint/article.md [hunk 6]
```diff
@@ -2,129 +2,129 @@
...
-In order to avoid confusion, it's not supported on bigints:
+برای جلوگیری از درهم شدن، این عملگر روی bigint ها پشتیبانی نمی شود.
 ```js run
 let bigint = 1n;
 
-alert( +bigint ); // error
+alert( +bigint ); // خطا
 ```
-So we should use `Number()` to convert a bigint to a number.
+باید ابتدا از `Number()` برای تبدیل یک bigint به نوع عددی استفاده کرد.
```
Comment (mahdyar) [inline 1-js/99-js-misc/05-bigint/article.md:59]: There is no `ابتدا` in the text, don't add your own words. Also, you missed `so`.
`پس باید از...`

File: 1-js/99-js-misc/05-bigint/article.md [hunk 7]
```diff
@@ -2,129 +2,129 @@
...
 
-Boolean operators, such as `||`, `&&` and others also work with bigints similar to numbers:
+عملگرهای منطقی نظیر `||` و `&&` روی bigint ها، مشابه نوع عددی (number) عمل می کنند:
 
 ```js run
-alert( 1n || 2 ); // 1 (1n is considered truthy)
+alert( 1n || 2 ); // 1 (1n برابر با true است)
 
-alert( 0n || 2 ); // 2 (0n is considered falsy)
+alert( 0n || 2 ); // 2 (0n برابر با false است)
```
Comment (mahdyar) [inline 1-js/99-js-misc/05-bigint/article.md:97]: `Considering` means `در نظر گرفتن`.

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by amirhshahbazi** [comment]
Thanks,
/done

**Review by mahdyar — APPROVED** [review]
Thank you, @amirhshahbazi!

**Pull request merged**